ledger-tool/src/bigtable.rs

```diff
@@ -584,6 +830,124 @@ impl BigTableSubCommand for App<'_, '_> {
+                                .takes_value(true)
+                                .help(
+                                    "Destination bigtable endpoint",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("destination_is_emulator")
+                            .long("destination-is-emulator")
+                            .value_name("DESTINATION_IS_EMULATOR")
+                            .takes_value(false)
+                            .help(
+                                "Destination is an emulator",
+                            ),
+                        )
+                        .arg(
+                            Arg::with_name("destination_instance_name")
+                                .long("destination-instance-name")
+                                .takes_value(true)
+                                .value_name("DESTINATION_INSTANCE_NAME")
+                                .default_value(solana_storage_bigtable::DEFAULT_INSTANCE_NAME)
+                                .help("Destination bigtable instance name")
+                        )
+                        .arg(
+                            Arg::with_name("destination_app_profile_id")
+                                .long("destination-app-profile-id")
+                                .takes_value(true)
+                                .value_name("DESTINATION_APP_PROFILE_ID")
+                                .default_value(solana_storage_bigtable::DEFAULT_APP_PROFILE_ID)
+                                .help("Destination bigtable app profile id")
+                        )
+                        .arg(
+                            Arg::with_name("starting_slot")
+                                .long("starting-slot")
+                                .validator(is_slot)
+                                .value_name("START_SLOT")
+                                .takes_value(true)
+                                .required(true)
+                                .help(
+                                    "Start copying at this slot",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("ending_slot")
+                                .long("ending-slot")
+                                .validator(is_slot)
+                                .value_name("END_SLOT")
+                                .takes_value(true)
+                                .help("Stop copying at this slot [default: starting-slot + 1]"),
+                        )
+                        .arg(
+                            Arg::with_name("force")
+                            .long("force")
+                            .value_name("FORCE")
+                            .takes_value(false)
+                            .help(
+                                "still upload whole block to destination although the block existed",
```

Comment:
```suggestion
                                "Force copy of blocks already present in destination Bigtable instance",
```

Reply:
Also, incidentally, it irritates me that we aren't consistent with our capitalization of "Bigtable" in clap help strings. Can you help pick a standard?

Reply:
sorry about that 😢 I made copy command prints "Bigtable" https://github.com/solana-labs/solana/pull/28122/commits/21fd88f7156c8f38c5b5fcaa0c3958d581b4c972